src/v/transform/transform_manager.cc

```diff
@@ -368,14 +389,15 @@ ss::future<> manager::create_processor(
           fut.get_exception());
         // Delay some time before attempting to recreate a processor
         // TODO: Should we have more sophisticated backoff mechanisms?
-        constexpr auto recreate_attempt_delay = 10s;
-        _queue.submit_delayed(
+        constexpr auto recreate_attempt_delay = 30s;
```

Comment:
intentional change (it's in a clock-type switch commit) ?

Reply:
No sorry, nice catch I'll move out to it's own commit

Reply:
[cb7f218](https://github.com/redpanda-data/redpanda/pull/13270/commits/cb7f218225849e6073f561d29e9f5dff86e8b890)